packages/atomic-react/package.json

```diff
@@ -30,7 +30,9 @@
     "commerce/"
   ],
   "dependencies": {
```

Comment:
We need to externalize lit to allow proper resolution by consumer (depending on Node or browser)